### DISCUSSION TITLE
Whatsapp iOS Analysis

### DISCUSSION THREAD

**Discussion by UCAPEM-ACADEMY** [sepinf-inc/IPED#2362] [General]
Friends, has someone found a way to extract WhatsApp domains in the new update of iOS 18 and 18.1, since with normal methods, the databases are not extracted and they can no longer be indexed with IPED, someone has managed to do it. ?

**Comment by lfcnassif** [comment]
Hi, what do you mean by domains?

**Reply by UCAPEM-ACADEMY** [reply]
Kind regards
I am referring to the directories that contain the application data

**Comment by wladimirleite** [comment]
> Friends, has someone found a way to extract WhatsApp domains in the new update of iOS 8 and 8.1, since with normal methods, the databases are not extracted and they can no longer be indexed with IPED, someone has managed to do it. ?

Hi! I guess you meant iOS 18 (not 8), right?!
We use forensic tools to extract data from iOS devices. 
Some of them already support iOS 18.x, and the others should support it soon.
I just extracted data from an iPhone with iOS 18.0, and IPED processed WhatsApp databases contained in the extraction successfully.

**Reply by UCAPEM-ACADEMY** [reply]
Greetings, it is indeed version 18, by mistake I did not complete the number.
Regardless of the paid acquisition tools, is there an opensource tool that can already extract the data from the new WhatsApp domains, which do you recommend?

**Reply by wladimirleite** [reply]
Unfortunately we only use commercial tools for extraction of mobile phones.
